website/public/js/filters/taskOrdering.js

```diff
@@ -22,6 +22,13 @@ angular.module('habitrpg')
       for (var i = 0; i < input.length; i++) {
         if (term.test(input[i].text) || term.test(input[i].notes)) {
           result.push(input[i]);
+        }else if (input[i].checklist) {
+          var checkListLen = input[i].checklist.length;
```

Comment:
I bet lodash has some magic function that'll do all the work of this loop for you.